metricbeat/module/windows/perfmon/doc.go

```diff
@@ -2,3 +2,6 @@
 Package perfmon collect windows performance counters.
 */
 package perfmon
+
+// go:generate go tool cgo -godefs defs_windows.go > defs_windows_amd64.go
```

Comment:
Please test this by running `go generate`.

From https://blog.golang.org/generate:

> The comment must start at the beginning of the line and have no spaces between the // and the go:generate.

The space after the slash needs to be removed otherwise the comment will not be executed.